config/config.go

```diff
@@ -470,6 +470,7 @@ type VTrack struct {
 
 type Event struct {
 	TimeoutMS int64 `mapstructure:"timeout_ms"`
+	Enabled   bool  `mapstructure:"enabled"`
```

Comment:
> @bretg I see you added requirement 2.1.19 to indicate when events should be considered enabled:
> 
> > 19. Events should be considered 'enabled' for a given account when any of the following conditions are true:
> >     a. host-level events are enabled and account-level events are enabled
> >     b. host-level events are enabled and the host-level default-account-config has events enabled.
> 
> If I'm not mistaken, from looking at the PBS-Java code and the discussion here, it appears PBS-Java's host-level events enabled flag is actually the host-level default-account-config events enabled flag. In other words, you don't have both a host flag and a default account config flag right? There's no need to have both.
> 
> @shahinrahbariasl PBS-Go actually does have the host-level account defaults config option so we could use that instead of adding this host-level flag. See field `AccountDefaults Account` in the `Configuration` struct in `config/config.go`.

Hi @bsardo, I was just addressing @guscarreon 's comments. You're right I can just do:
```
account_defaults:
  events_enabled: true
```
and that should enable events by default. Didn't know about that flag, I guess there's no need for this PR then...

Reply:
Very well, updated the doc to remove references to host-level config, clarifying that it's host-level default-account-config